.gitignore

```diff
@@ -5,3 +5,6 @@ package-lock.json
 
 package.json
 
+
+
```

Comment:
Kindly fix your linters errors and make sure that every linters is passing before requesting any review.
![image](https://user-images.githubusercontent.com/24703171/158500868-0d8c1428-d597-4945-8edf-b4aeb73bb13e.png)
The `stylelint` is not passing, Kindly fix this problem.